postgres-driver/blockchain.go

```diff
@@ -0,0 +1,100 @@
+package postgresdriver
+
+import (
+	"database/sql"
+
+	"github.com/lib/pq"
+)
+
+const (
+	selectAllBlockchainsScript = "SELECT * FROM blockchains"
+	selectBlockchainByID       = "SELECT * FROM blockchains WHERE blockchain_id = $1"
+)
+
+type dbBlockchain struct {
+	ID                int            `db:"id"`
+	BlockchainID      string         `db:"blockchain_id"`
+	Altruist          sql.NullString `db:"altruist"`
+	Blockchain        sql.NullString `db:"blockchain"`
+	BlockchainAliases pq.StringArray `db:"blockchain_aliases"`
+	ChainID           sql.NullString `db:"chain_id"`
+	ChaindIDCheck     sql.NullString `db:"chain_id_check"`
+	Description       sql.NullString `db:"description"`
+	Index             sql.NullInt64  `db:"_index"`
+	LogLimitBlocks    sql.NullInt64  `db:"log_limit_blocks"`
+	Network           sql.NullString `db:"network"`
+	NetworkID         sql.NullString `db:"network_id"`
+	NodeCount         sql.NullInt64  `db:"node_count"`
+	Path              sql.NullString `db:"_path"`
+	RequestTimeout    sql.NullInt64  `db:"request_timeout"`
+	Ticker            sql.NullString `db:"ticker"`
+}
+
+func (b *dbBlockchain) toBlockchain() *Blockchain {
+	return &Blockchain{
+		BlockchainID:      b.BlockchainID,
+		Altruist:          b.Altruist.String,
+		Blockchain:        b.Blockchain.String,
+		BlockchainAliases: b.BlockchainAliases,
+		ChainID:           b.ChainID.String,
+		ChaindIDCheck:     b.ChaindIDCheck.String,
+		Description:       b.Description.String,
+		Index:             b.Index.Int64,
+		LogLimitBlocks:    b.LogLimitBlocks.Int64,
+		Network:           b.Network.String,
+		NetworkID:         b.NetworkID.String,
+		NodeCount:         b.NodeCount.Int64,
+		Path:              b.Path.String,
+		RequestTimeout:    b.RequestTimeout.Int64,
+		Ticker:            b.Ticker.String,
+	}
+}
+
+// Blockchain struct handler representing a Blockchain
+type Blockchain struct {
```

Comment:
Same here as the previous comment, i.e. could we avoid defining new types for the postgres-driver package, and let this package return types declared by the `repository` package.